### PR TITLE
Adding authentication info to the migrations API

### PR DESCRIPTION
**Pull request by Chuan-Ru**

### CODE REVIEW

File: content/v3/migration/migrations.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ title: Migrations
 
 {:toc}
 
+The Migrations API *is only available to [authenticated](https://developer.github.com/v3/#authentication) organization owners*. Non-organization owners will receive a `404` response if they try to access it.
```
Comment (gjtorikian) [inline content/v3/migration/migrations.md]: Can you wrap this in a tip block, like:

```
{{#tip}}

...

{{/tip}}
```

Comment (gjtorikian) [inline content/v3/migration/migrations.md]: Ah, I'd also suggest making the link relative: `[authenticated](/v3/#authentication) organization...`

**Comment by gjtorikian** [comment]
Looks pretty good to me, I made a few comments.

**Comment by Chuan-Ru** [comment]
^^

File: content/v3/migration/migrations.md (hunk 1 above)
Comment (Chuan-Ru) [inline content/v3/migration/migrations.md]: ??

Comment (Chuan-Ru) [inline content/v3/migration/migrations.md]: = =!

Comment (Chuan-Ru) [inline content/v3/migration/migrations.md]: @@!

Comment (Chuan-Ru) [inline content/v3/migration/migrations.md]: .........

**Pull request closed without merging**